<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Txt 2 Epub</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="le_txt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_author">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_reg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>^\s*([第卷][0123456789一二三四五六七八九十零〇百千两]*[章回部节集卷].*)\s*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_epub">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_image">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cover.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>作 者</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_epub">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>160</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_txt">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>40</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>30</y>
      <width>30</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输 入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>书 名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>章 节</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>输 出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_convert">
       <property name="text">
        <string>转   换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>      </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_dir">
       <property name="text">
        <string>目   录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>      </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_reset">
       <property name="text">
        <string>重   置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>      </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cover">
       <property name="text">
        <string>封   面</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
